--- config	2022-07-04 09:12:13.303374423 +0100
+++ config	2022-07-04 09:12:33.230042167 +0100
@@ -17,0 +18 @@
+        "bluetooth",
@@ -135,0 +137,10 @@
+    },
+    "bluetooth": {
+        "format": "<span foreground='#928374'></span>",
+        "format-connected": "<span foreground='#928374'></span> {device_alias}",
+        "format-connected-battery": "<span foreground='#928374'></span> {device_alias} {device_battery_percentage}%",
+        "tooltip-format": "{controller_alias}\t{controller_address}\n\n{num_connections}connected",
+        "tooltip-format-connected": "{controller_alias}\t{controller_address}\n\n{num_connections} connected\n\n{device_enumerate}",
+        "tooltip-format-enumerate-connected": "{device_alias}\t{device_address}",
+        "tooltip-format-enumerate-connected-battery": "{device_alias}\t{device_address}\t{device_battery_percentage}%",
+        "on-click": "rofi-bluetooth"
