--- config.desktop	2022-06-25 13:56:22.470249480 +0100
+++ config.laptop	2022-06-25 13:51:15.110238304 +0100
@@ -17,0 +18 @@
+        "bluetooth",
@@ -136,0 +138,10 @@
+    "bluetooth": {
+        "format": "<span foreground='#928374'></span>",
+        "format-connected": "<span foreground='#928374'></span> {device_alias}",
+        "format-connected-battery": "<span foreground='#928374'></span> {device_alias} {device_battery_percentage}%",
+        "tooltip-format": "{controller_alias}\t{controller_address}\n\n{num_connections}connected",
+        "tooltip-format-connected": "{controller_alias}\t{controller_address}\n\n{num_connections} connected\n\n{device_enumerate}",
+        "tooltip-format-enumerate-connected": "{device_alias}\t{device_address}",
+        "tooltip-format-enumerate-connected-battery": "{device_alias}\t{device_address}\t{device_battery_percentage}%",
+        "on-click": "rofi-bluetooth"
+    },
@@ -143 +154,2 @@
-        "tooltip-format": "{ifname}: {ipaddr}/{cidr}\n {bandwidthDownBits}\n {bandwidthUpBits}"
+        "tooltip-format": "{ifname}: {ipaddr}/{cidr}\n {bandwidthDownBits}\n {bandwidthUpBits}",
+        "on-click": "rofi-wifi-menu"
