--- config	2022-11-20 06:17:43.409315888 +0000
+++ config	2022-11-20 08:58:19.523406534 +0000
@@ -17,0 +18 @@
+        "bluetooth",
@@ -139,0 +141,10 @@
+    },
+    "bluetooth": {
+        "format": "<span foreground='#928374'></span>",
+        "format-connected": "<span foreground='#928374'></span> {device_alias}",
+        "format-connected-battery": "<span foreground='#928374'></span> {device_alias} {device_battery_percentage}%",
+        "tooltip-format": "{controller_alias}\t{controller_address}\n\n{num_connections}connected",
+        "tooltip-format-connected": "{controller_alias}\t{controller_address}\n\n{num_connections} connected\n\n{device_enumerate}",
+        "tooltip-format-enumerate-connected": "{device_alias}\t{device_address}",
+        "tooltip-format-enumerate-connected-battery": "{device_alias}\t{device_address}\t{device_battery_percentage}%",
+        "on-click": "rofi-bluetooth"
